frmts/mem/memdataset.cpp

```diff
@@ -2483,6 +2483,9 @@ void GDALRegister_MEM()
 "   </Option>"
 "</CreationOptionList>" );
 
+    poDriver->SetMetadataItem( GDAL_DCAP_FIELD_DOMAINS, "YES" );
```

Comment:
oops I merged this, but I now realize that this part is incorrect. This should be moved to the ogr/ogrsf_frmts/mem/ driver.  I'll do the fix in a follow up commit

Reply:
Done in 7da4d8b0b0

Reply:
Thanks!